src/Docker.PowerShell/project.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "0.1.0-alpha",
+  "version": "0.0.0-alpha",
```

Comment:
Are you only going to put real revs in the project.json files when an actual tag is produced then?